fw/http_stream.c

```diff
@@ -409,34 +477,85 @@ do {									\
 						SET_STATE(new_state);
 					}
 				}
+			} else {
```

Comment:
You do not process a case when we receive EN_OF_STREAM without end of headers here. Please also add appropriate test for this case

Reply:
This case is handled in parser, because it's valid case for trailers. Moving it here doesn't have sense.

Reply:
I wrote test and check error.

Reply:
We also need check case when client send us two or more END_STREAM flags, and return error

Reply:
Oh I see, you mean CONTINUATION frames. By the protocol [CONTINUATION](https://www.rfc-editor.org/rfc/rfc9113.html#name-continuation) frame doesn't have END_STREAM flag. In previous versions of Tempesta we handled such frames correctly, but it's incorrect we must treat such cases as protocol error.